src/helpers/channels/clone_channel.ts

```diff
@@ -0,0 +1,28 @@
+import { cacheHandlers } from "../../cache.ts";
+import { createChannel } from "./create_channel.ts";
+import { CreateGuildChannel } from "../../types/guilds/create_guild_channel.ts";
+import { DiscordenoChannel } from "../../structures/channel.ts";
+
+/** Create a copy of a channel */
+export async function cloneChannel(channelId: string, reason?: string) {
+  const channelToClone: DiscordenoChannel | undefined = await cacheHandlers.get(
```

Comment:
Don't use explicit types if cacheHandlers.get already has return types
```suggestion
  const channelToClone = await cacheHandlers.get(
```